src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -100,15 +100,14 @@
      * Compile sets the required command line arguments and runs
      * native-image
      *
-     * @param cp The classpath of the project to be run
      * @return true if the process ends successfully, false otherwise
      * @throws IOException
      * @throws InterruptedException
      */
     @Override
-    public boolean compile(String cp) throws IOException, InterruptedException {
-        cp = processClassPath(cp);
-        extractNativeLibs(cp);
+    public boolean compile() throws IOException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'compile()' has an NPath complexity of 5184](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5152426)